src/test/scala/is/hail/methods/AggregatorSuite.scala

```diff
@@ -123,6 +126,31 @@ class AggregatorSuite extends SparkSuite {
     p.check()
   }
 
+  @Test def testMaxMin() {
+    val rdd1 = sc.parallelize(Seq(
+      Annotation("a",  0, null, -1, -1.0, null),
+      Annotation("a", -1, null, -2, 1.0, null)), numSlices = 1)
+
+    val rdd2 = sc.parallelize(Seq(
+      Annotation("a",  1, -1, -3, null, null),
+      Annotation("a",  0, -2, -4, 0.0, null)), numSlices = 1)
+
+    val rdd = rdd1.union(rdd2)
```

Comment:
Two per partition because I wanted to test null behavior inside both combOp and seqOp. I'm happy to change to one RDD, but I felt this approach made it transparent in code what was happening.